go.sum

```diff
@@ -0,0 +1,123 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
is this file up-to-date?

Reply:
./hack/verify-deps.sh will fail if it's not